visits-service-new/src/test/java/com/petclinic/visits/visitsservicenew/PresentationLayer/VisitControllerUnitTest.java

```diff
@@ -50,7 +50,7 @@ class VisitControllerUnitTest {
 
 
     Set<SpecialtyDTO> set= new HashSet<>();
-
+    Set<Workday> workdaySet = new HashSet<>();
```

Comment:
Smart value change Karina. Modifications made to visit-service-new look good as well